torrt/utils.py

```diff
@@ -63,6 +63,22 @@ def dump_contents(filename, contents):
         f.write(text)
 
 
+def base64encode(string_or_bytes):
+    """Return base64 encoded input
+
+    :param string_or_bytes:
+
+    """
+
+    if six.PY3:
```

Comment:
No reason to check it on every execution. Let's define a proper function on import.

```
if six.PY3:
    def base64encode(string_or_bytes):
        ...
else:
    def base64encode(string_or_bytes):
        ...
```

Import on every call wouldn't cause huge performance difference, but could be avoided still.